src/containers/PermissionsApplication.jsx

```diff
@@ -57,7 +57,13 @@ const PermissionsApplication = ({ match }) => {
           ).map(([key, value]) => (
             <Link to={`/permissions/${appName}/${key}`} key={key}>
               <Typography variant="h4">
-                {value.type} : {value.verbs ? value.verbs.join(' / ') : 'ALL'}
+                {value.type} :{' '}
+                {!value.verbs ||
+                (value.verbs.length > 1 && value.verbs.indexOf('GET') >= 0)
+                  ? 'Lecture et Écriture'
+                  : value.verbs.length === 1 && value.verbs[0] === 'GET'
```

Comment:
let's use https://www.w3schools.com/Jsref/jsref_includes.asp here 🙏

Reply:
Because if ```length === 1 && value.verbs.indexOf('GET') >= 0```, it is not 'Lecture et Écriture' but only 'Lecture'